PR: https://github.com/odoo/odoo/pull/48617

From: 6dc283581488b2fd22a17bd4aaf3b1c228c7033d
From: Thomas Carlier
Date: 2020-03-30 14:08:07

Structural Changes: 3
Total Changes: 17

[FIX] hr_expense: remove the expense_extract options from hr_expense

before this commit:
- the selection field with OCR options was in hr_expense module

after this commit:
- the selection field is now in expense_extract module

closes odoo/odoo#48617

X-original-commit: 9440139105825d4efa5fda23149c60179870f9f3
Related: odoo/enterprise#9553
Signed-off-by: Christophe Simonis <chs@odoo.com>
Signed-off-by: Thomas Carlier <Explicitapps@users.noreply.github.com>

================================= pseudo patch: =================================

--- a/addons/hr_expense/models/__init__.py
+++ b/addons/hr_expense/models/__init__.py
@@ -7,4 +7,3 @@ from . import hr_expense
 from . import product_template
 from . import res_config_settings
 from . import account_journal_dashboard
-from . import res_company

--- a/addons/hr_expense/models/res_company.py
+++ b/None
@@ -1,14 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-
-class ResCompany(models.Model):
-    _inherit = 'res.company'
-
-    expense_extract_show_ocr_option_selection = fields.Selection([
-        ('no_send', 'Do not digitalize bills'),
-        ('manual_send', "Digitalize bills on demand only"),
-        ('auto_send', 'Digitalize all bills automatically')], string="Send mode on expense attachments",
-        required=False, default='auto_send')

--- a/addons/hr_expense/models/res_config_settings.py
+++ b/addons/hr_expense/models/res_config_settings.py
@@ -12,8 +12,6 @@ class ResConfigSettings(models.TransientModel):
 
     module_hr_payroll_expense = fields.Boolean(string='Reimburse Expenses in Payslip')
     module_hr_expense_extract = fields.Boolean(string='Send bills to OCR to generate expenses')
-    expense_extract_show_ocr_option_selection = fields.Selection(related='company_id.expense_extract_show_ocr_option_selection',
-        string='Expense processing option', readonly=False)
 
 
     @api.model
